src/audio/dai.c

```diff
@@ -799,7 +799,7 @@ static int dai_config(struct comp_dev *dev, struct sof_ipc_dai_config *config)
 		}
 	}
 
-	return 0;
+	return dai_set_config(dd->dai, config);
```

Comment:
I assume there is something else ensuring dai_config is called after the IPC handler finishes everything it wants to do?

Reply:
@paulstelian97 `dai_config` is always called. The only reason it would fail is because the IPC message itself would be somehow messed up.